src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -810,7 +790,21 @@ MulticopterAttitudeControl::task_main()
 			vehicle_status_poll();
 			vehicle_motor_limits_poll();
 
+//			/* override ACRO mode when both roll and pitch < 70% */
```

Comment:
This is how I thought the code should work on first inspection, since att_control has a flag called "control_attitude_enabled".  But (surprisingly) the attitude setpoint is not generated in att_control... instead it is generated and published by mc_pos_control. My understanding of this partitioning of the code is that it is the lesser of two evils, hence the comment so I don't waste time when I next work on something similar.